conda/recipes/librmm/meta.yaml

```diff
@@ -97,7 +98,7 @@ outputs:
     script: install_librmm_tests.sh
     build:
       number: {{ GIT_DESCRIBE_NUMBER }}
-      string: cuda{{ cuda_major }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+      string: cuda{{ cuda_major }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}{{ date_string }}
```

Comment:
I think `date_string` needs to be before `GIT_DESCRIBE_HASH`. I believe the way conda chooses a package is by ordering the the build string alphabetically, then picking the latest. So we want the date to sort the packages since the hash is basically random.

Reply:
Fixed in 013cc13 and 2ba4d02